packages/static/src/pagination/styles/pagination.module.css

```diff
@@ -0,0 +1,78 @@
+@import '../../theme/theme.module.css';
+
+/* LEVEL1 */
+
+.pagination {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: var(--cdg-spacing-4);
+  padding: var(--cdg-spacing-2) var(--cdg-spacing-6);
+  flex-wrap: wrap;
+}
+
+.paginationItemCounting {
+  font-weight: 600;
+}
+
+.paginationRowsCounting {
+  font-weight: 600;
+  display: flex;
+  white-space: nowrap;
+  margin-right: auto;
```

Comment:
why would you need margin-right when you use flex?

Reply:
display flex for horizontal element and margin-right: auto for create space between rowCounting component and pagin component